documentation/en/cli-lotus-miner.md

```diff
@@ -77,13 +82,15 @@ USAGE:
    lotus-miner init restore [command options] [backupFile]
 
 OPTIONS:
-   --nosync                don't check full-node sync status (default: false)
-   --config value          config file (config.toml)
-   --storage-config value  storage paths config (storage.json)
-   --help, -h              show help
+   --nosync                       don't check full-node sync status (default: "false")
```

Comment:
5. `"false"` vs `false` - figure out what the quoting rule is, could be strings only?

Reply:
Fixed.